server/app.js

```diff
@@ -1,12 +1,13 @@
 const express = require("express");
-const { getCategories, getReviews, getReviewById } = require("./controllers/controllers.games");
+const { getCategories, getReviews, getReviewById, getCommentsByReviewId } = require("./controllers/controllers.games");
```

Comment:
Might be worth splitting these controllers/models out to seperate files to keep each function seperate.